src/config/dbConfig.ts

```diff
@@ -0,0 +1,9 @@
+const dbConfig = {
+  user: 'your_username',
+  host: 'your_host',
+  database: 'your_database',
+  password: 'your_password',
+  port: 'your_port_number',
+};
+
+export default dbConfig;
```

Comment:
It'll be better to set these up using env variables. So we don't have to edit this file on every local machine and the server differently and do the dance of not committing the file when pushing changes. 

To ensure we don't run into run time issues we can configure this object on the server start let that process fail if the env variables are not properly set.

Reply:
@Piumal1999 @anjula-sack